win32ss/user/winsrv/usersrv/init.c

```diff
@@ -121,8 +121,32 @@ CreateSystemThreads(PVOID pParam)
 
 CSR_API(SrvCreateSystemThreads)
 {
-    DPRINT1("%s not yet implemented\n", __FUNCTION__);
-    return STATUS_NOT_IMPLEMENTED;
+    HANDLE ServerThread;
+    CLIENT_ID ClientId;
+    NTSTATUS Status;
+
```

Comment:
Using a sub-structure inside ApiMessage, this function will have to determine where to create the thread: in CSRSS itself, or in a remote process.